dotnet/src/InternalUtilities/src/Http/HttpContentExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.IO;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.SemanticKernel.Diagnostics;
+
+/// <summary>
+/// Provides extension methods for working with HTTP content in a way that translates HttpRequestExceptions into HttpOperationExceptions.
+/// </summary>
+internal static class HttpContentExtensions
```

Comment:
nit: Same comment regarding naming as above for methods in this class.
I would name it something like `ReadAsStringWithExceptionMapping`, but again, it's just my personal preference, feel free to resolve this comment.

Reply:
Renamed